### PR TITLE
Update Layout05.svg to Inherit Height from parent

### PR DESCRIPTION
**Pull request by DZ-FSDev**

### CODE REVIEW

File: ViewPortFluidDesign/image/Layout05.svg [hunk 1]
```diff
@@ -21,16 +21,15 @@ foreignObject {
...
-
-  .header-container {width: 50%; height: 10%; background-color: #3e2723; margin: 0 auto;}
-  .banner-container {width: 50%; height: 30%; background-color: #4e342e; margin: 0 auto;}
-  .content-container {width: 50%; height: 50%; background-color: #5d4037; margin: 0 auto;}
-  .footer-container {width: 50%; height: 10%; background-color: #6d4c41; margin: 0 auto;}
+  #wrap {font-size: 4vh; height:100%; color: #fff; text-align: center; text-transform: uppercase;}
+  #header-wrap {height: 10%; background-color: #8d6e63;}
+  #banner-wrap {height: 30%; background-color: #a1887f;}
+  #content-wrap {height: 50%; background-color: #bcaaa4;}
+  #footer-wrap {height: 10%; background-color: #d7ccc8;}
```
Comment (DZ-FSDev) [inline ViewPortFluidDesign/image/Layout05.svg:28]: These heights add up to 100% and are children of `#wrap`

File: ViewPortFluidDesign/image/Layout05.svg [hunk 2]
```diff
@@ -21,16 +21,15 @@ foreignObject {
...
+  .header-container {width: 50%; height: 100%; background-color: #3e2723; margin: 0 auto;}
+  .banner-container {width: 50%; height: 100%; background-color: #4e342e; margin: 0 auto;}
+  .content-container {width: 50%; height: 100%; background-color: #5d4037; margin: 0 auto;}
+  .footer-container {width: 50%; height: 100%; background-color: #6d4c41; margin: 0 auto;}
```
Comment (DZ-FSDev) [inline ViewPortFluidDesign/image/Layout05.svg:32]: These are each children of their respective wrap <div> elements so they can take 100% of the height given by their parent.

**Review by DZ-FSDev — COMMENTED** [review]
Related to #2

**Comment by DevSamLee** [comment]
Thanks, I got it!

**Pull request merged**